### PR TITLE
update fr site new value

### PR DESCRIPTION
**Pull request by zhanw15**

### CODE REVIEW

File: i18n/fr.toml [hunk 1]
```diff
@@ -118,7 +118,8 @@ other = "Utilisateurs"
 [docs_label_i_am]
 other = "JE SUIS..."
 
-
+[examples_heading]
+other = "Exemple"
```
Comment (sftim) [inline i18n/fr.toml:122]:
```suggestion
other = "Exemples"
```

**Comment by sftim** [comment]
Please rebase this pull request against _master_ to bring in a work-around for issue #22222.

**Comment by kbhawkey** [comment]
Hello @zhanw15 . Thanks for this update.
Are you able to squash your changes to a single commit?
Thank you.

**Pull request merged**